src/main/kotlin/assertk/assertions/path.kt

```diff
@@ -0,0 +1,56 @@
+package assertk.assertions
+
+import assertk.Assert
+import assertk.assertions.support.expected
+import assertk.assertions.support.show
+import java.nio.file.Files
+import java.nio.file.LinkOption
+import java.nio.file.Path
+
+fun Assert<Path>.isRegularFile(vararg options: LinkOption) {
```

Comment:
Can you put the path assertions in a separate pr? That way one doesn't block the other.

Reply:
Also, path isn't in android until api 26, we might want a separate java8 artifact.

Reply:
Regarding separate PRs: How do I do that now?
Regarding separate java8 artiffact: I have even less clue how to do that.

Reply:
Don't worry about the java8 artifact, still thinking about the right solution there. For a separate pr, you can branch off this one and delete the inputstream files, then delete the path files from here.